src/main.cpp

```diff
@@ -39,9 +39,12 @@ void setup()
   ahrs.averageQuaternion(initialQuaternion);             // determine initial attitude
   ahrs.averageAngularVelocities(initialAngularVelocity); // determine initial attitude
   buzz(2);                                               // 2 beeps => AHRS setup complete
+  Serial.println("waiting for PiSerial...");
```

Comment:
Instead of `Serial.println` use our `logSerial` with a versobity level of either `debug` or `info`.

Reply:
Done